includes/Webhook/Hooks/OrderStatusWebhook.php

```diff
@@ -84,6 +86,21 @@ private function isPrinted($order): bool
         );
     }
 
+    private function updateOrderBarcode(Order $order): void
+    {
+        $orderData      = get_post_meta($order->getExternalIdentifier(), WCMYPA_Admin::META_PPS, true);
+        $orderShipments = $order->getOrderShipments();
+        $barcode        = $orderShipments[0]['shipment']['barcode'];
```

Comment:
Als er geen shipments zijn, zal dit volgens mij een fout geven. Weten we 100% zeker dat hier altijd 1 shipment zal zijn? En is het altijd het juiste shipment dan, moeten we niet de laatste kiezen?